### ISSUE TITLE
Underlining of links

### ISSUE THREAD

**Issue by michaelwood** [ThreeSixtyGiving/360-ds#59]
By default all links in the 360-ds have been designed to have no underline which may be an accessibility/inclusion issues

https://github.com/ThreeSixtyGiving/360-ds/blob/master/src/scss/pre/_reset.scss#L133

**Comment by michaelwood** [comment]
We're going to make links in paragraphs underlined by default

**Comment by michaelwood** [comment]
Approved to remove the text-decoration: none for any link in the design system.

**Comment by michaelwood** [comment]
@mariongalley can we bump this up to be a "within a few days" on the project board?

**Comment by mariongalley** [comment]
@michaelwood Yes!

**Comment by codemacabre** [comment]
Just to double check @mariongalley @michaelwood does this include links in the top-bar (left) and footer (right)?
<img src="https://user-images.githubusercontent.com/49149591/201647283-7b86a3ce-ce3b-4462-ba23-b7aed80f7fdf.png" width="300" alt="screenshot of top bar menu, with underlined links" /> <img src="https://user-images.githubusercontent.com/49149591/201647515-762e1199-9b80-4500-810b-a375aeb1dbb8.png" width="300" alt="screenshot of footer, with underlined links" />

**Comment by mariongalley** [comment]
Good question, I'd say so yes. I can't see what you mean by top-bar (left), I can only see some right-aligned links in the top bar and they already appear underlined to me.

As for the footer, it does seem arbitrary that some of the links are underlined and others not as it currently stands:
![image](https://user-images.githubusercontent.com/82958988/201651014-e8bd0a58-343f-4456-830c-1551e7db97cd.png)

**Comment by codemacabre** [comment]
Ah sorry, I meant image left and image right. The two screenshots I posted are with underlines added. Thanks for the clarification; I'll adjust to underline as in my screenshots.

**Comment by mariongalley** [comment]
This looks fixed @codemacabre

**Comment by drkane** [comment]
@mariongalley - I'm just looking at updating the main website to the latest styes. This change produces quite a lot of underlining on the main page where previously there wasn't any - we may need to explicitly not underline the front page boxes and the tags on the blog posts as they're the most jarring.

![Screenshot 2023-02-22 at 15-09-44 360Giving - 360Giving](https://user-images.githubusercontent.com/1049910/220664801-9d199433-a574-4e52-8f38-7c4019402d53.png)

**Comment by mariongalley** [comment]
@drkane I agree it looks a bit odd here, but in general I do think it was an issue that links weren't underlined, and it wasn't always obvious which things were clickable as a result. I personally don't mind it on the blogs tags, but I don't think it looks good on the cards which you can tell are clickable from the interaction on mouseover.

**Comment by mariongalley** [comment]
Are you happy to exclude it from the boxes?

**Comment by drkane** [comment]
Cool - I'm also a big fan of underlining links. I'll exclude from the boxes and then we can see what it looks like.
